config/initializers/elasticsearch.rb

```diff
@@ -0,0 +1,8 @@
+require "elasticsearch"
+
+SearchClient = Elasticsearch::Client.new(
```

Comment:
[should we log in development?](https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport#logging)

Reply:
I'm the last projects where I used ES with Rails I toggled it via an environment variable. It had the default of `true` in development and `false` in production. Being able to toggle it on in production without a deployment was occasionally useful though.

```
log: ENV['RAILS_ES_LOGGING_ON'].to_bool
```

Nvm the `.to_bool` helper, it's just something I add to some projects for easier env var handling:

```ruby
class String
  def to_bool
    return true if self =~ /^(true|t|yes|y|1)$/i
    return false if blank? || self =~ /^(false|f|no|n|0)$/i
    fail ArgumentError, "invalid value for Boolean: \"#{self}\""
  end
end
```

Reply:
Going to go with the usual `Rails.env.development?` since we have DD tracing and Honeycomb and will have logs via ElasticCloud so I think the chances of needed Rails production logging are low